app/javascript/src/components/Dashboard/constants.js

```diff
@@ -0,0 +1,3 @@
+export const SINGULAR = { count: 1 };
+
+export const PLURAL = { count: 2 };
```

Comment:
Let's keep these in global constants as it might've to be used across multiple components.